kythe/java/com/google/devtools/kythe/extractors/java/standalone/AbstractJavacWrapper.java

```diff
@@ -122,18 +121,25 @@ private static void outputIndexInfo(CompilationDescription indexInfo) throws IOE
       if (outputFile.endsWith(IndexInfoUtils.KZIP_FILE_EXT)) {
         IndexInfoUtils.writeKzipToFile(indexInfo, outputFile);
       } else {
-        IndexInfoUtils.writeKindexToFile(indexInfo, outputFile);
+        System.err.printf("Unsupported output file: %s%n", outputFile);
```

Comment:
%n -> \n ?

Reply:
I'm unsure here.  The rest of this file is already using %n, so I'd like to keep it that way for now.  If we want to move it all to \n, rather do that as a separate refactor step, and all at once (and maybe other files as well?).

In any event, %n is more platform-agnostic, I think?  (Windows?)

Reply:
Yup ok, from the java docs, %n is the platform-agnostic one

https://docs.oracle.com/javase/tutorial/java/data/numberformat.html

Reply:
TIL